### ISSUE TITLE
XML namespace attributes bug

### ISSUE THREAD

**Issue by trevordixon** [inikulin/whacko#8]
$ = whacko.load('<svg><use xlink:href="#abcd"></use></svg>')
    $('use').attr('href') === '#abcd' // should be undefined
    $('use').attr('xlink:href') === undefined // should be '#abcd'

This isn't an issue in Cheerio, but http://stackoverflow.com/questions/22081551/svg-image-turns-black-after-updating-his-path-via-jquery makes me think it might have been at some point.

**Comment by inikulin** [comment]
Yes, this is not the way it supposed to work according to DOM spec. Thank you for the report.
